src/ontology/foodon-edit.owl

```diff
@@ -65876,7 +65896,10 @@ SubClassOf(obo:FOODON_03530219 obo:FOODON_03530218)
 
 # Class: obo:FOODON_03530220 (food packing medium not drained)
 
+AnnotationAssertion(obo:IAO_0000112 obo:FOODON_03530220 "A can of chickpeas where the liquid medium has not been drained."@en)
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03530220 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03530220 "A food object quality where the medium in which the food is packed is still surrounding the food."@en)
+AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530220 "0000-0001-8008-8249"@en)
```

Comment:
Add ORCiD PURL prefix and don't include an `@en` tag (probably an accidental carry over form a previous annotation).

```suggestion
AnnotationAssertion(obo:IAO_0000117 obo:FOODON_03530220 <http://orcid.org/0000-0001-8008-8249>)
```